docs/system-admin-guide/README.md

```diff
@@ -16,15 +16,18 @@ If you are an administrator on your OpenProject environment,  you can access the
 
 Click on one of the categories in order to proceed with the configuration.
 
-![System-administration](System-administration.png)
+![image-20201006154046435](image-20201006154046435.png)
+
+
 
 ## Overview
 
-| Popular Topics                                     | Description                                                  |
+| Topic                                              | Content                                                      |
 | -------------------------------------------------- | :----------------------------------------------------------- |
 | [Users and permissions](./users-permissions)       | How to manage users, groups and permissions in OpenProject?  |
 | [Work packages](./manage-work-packages)            | How to configure work packages, types, status and workflows? |
 | [Custom fields](./custom-fields)                   | Set custom fields for work packages, Spent time, projects, versions, users, groups and more. |
+| [Attribute help texts](/attribute-help-texts)      | Add help texts to explain custom fields in projects and work packages. |
```

Comment:
I would change this to e.g. "explain attributes, including custom fields" (since attribute help texts are also available for standard attributes).